### ISSUE TITLE
Dashboard inaccessible 

### ISSUE THREAD

**Issue by pabogdan** [rtfd/readthedocs.org#1589]
Monday morning at around 9AM BST the dashboard became inaccessible due to an unknown reason.
According to @agjohnson (chat on IRC) the queue server went down. During this downtime, the language of the error page would change randomly after every refresh. 

Actual documentation pages seem to still be up though.

After reporting the initial issue, everything started working as normal for about half an hour, then everything went dark again. (currently 11AM BST)

Console error:
```bash
Failed to load resource: <dashboard link> the server responded with a status of 500 (OK)
```

UPDATE 1: Pushing a commit to the repo that has docs linked to readthedocs also takes down the documentation pages

**Comment by eivind88** [comment]
I'm having the same issue.
Currently only seeing a fail whale.

![skjermbilde 2015-08-24 kl 16 36 57](https://cloud.githubusercontent.com/assets/939844/9442888/64772388-4a7e-11e5-997b-c757f3def4c1.png)

**Comment by davehunt** [comment]
I'm seeing this too.

**Comment by agjohnson** [comment]
I noticed our session caching went belly up and bounced the cache last night. About 30m later, the cache crashed again, with an OOM state, but this time the cache wasn't consuming any memory. I'm still poking this, and why I didn't get woken up

**Comment by agjohnson** [comment]
Also, pushing a repo to us when our build queue is hosed shouldn't remove the current documentation. I'm going to open up another ticket to audit this.

**Comment by pabogdan** [comment]
@agjohnson Sorry I forgot to update the issue, the take down of the docs was just temporary, they came back up the next time I checked

**Comment by agjohnson** [comment]
@pabogdan at least they didn't go completely down then. We should probably still be ensuring we don't touch them at all until a successful build.

Also, I've discovered the original culprit here. The very checks we used to monitor the cache turned on us in a thundering herd and caused an out of memory state. I'll address in our monitoring and close out this issue.

Thanks for the report @pabogdan et al!
